### DISCUSSION TITLE
House load/consumption

### DISCUSSION THREAD

**Discussion by michaelschwarz** [wills106/homeassistant-solax-modbus#474] [Q&A]
I am still quite new to Solax and HA. What I am looking for is the total house load (all devices, plugs...), without battery charging or inverter side consumption, also without grid export.

What value is representing that?

**Comment by wills106** [comment]
The closest you will get without further work is "House Load Alt" which is disabled by default, but you can enable it.

**Reply by michaelschwarz** [reply]
Is there any good documentation available on each of the fields? I have the Solax PDF Modbus overview, but it would be interesting to also know what are the different totals or yield fields.

The House Load Alt I found and enabled it, and have a value already. The House Load was approx. 300 W higher before. I have around 200-250 W during the night, but with House Load around 500-600 W.
